.github/bin/run-clang-tidy-cached.cc

```diff
@@ -83,6 +83,24 @@ fs::path GetCacheDir() {
   return fs::path(getenv("TMPDIR") ?: "/tmp");
 }
 
+// Fix filename paths that are not emitted relative to project root.
```

Comment:
Perhaps include a few example transformation in comments?

Reply:
Good idea. Added a comment at the top.